src/analysisd/analysisd.c

```diff
@@ -984,7 +986,10 @@ void OS_ReadMSG_analysisd(int m_queue)
     }
 
     /* Initialize EPS limits */
-    load_limits(Config.eps.maximum, Config.eps.timeframe, Config.eps.maximum_found);
+    analysisd_limits = init_limits(Config.eps.maximum, Config.eps.timeframe);
+    if (!analysisd_limits->enabled && !Config.eps.maximum_found && analysisd_limits->timeframe > 0) {
+            mwarn("The eps maximum value is missing in the configuration block.");
```

Comment:
Several doubts here:

- This is a warning? Because I don't see the limits section in the config template, so I think this warning would appear always by default.

- If the limits have values by default we should apply these values if we don't find it configured so this warning should not appear. If we are not applying limits by default, we cannot continue the execution with null values.

Reply:
Done!